src/app/ad-authoring/ad-authoring.state.ts

```diff
@@ -9,15 +9,177 @@ export interface AdAuthoringWorkflowState {
   readonly callToAction?: CallToActionEnum;
   readonly fileSrc?: string;
   readonly file?: File;
+  readonly AMPHTMLstring: string;
 }
-// add a default state for the fields above
+
 @Injectable({
   providedIn: 'root',
 })
 export class AdAuthoringWorkflowStateContainer {
+  generateAMPHTML(
```

Comment:
As we discussed offline, let's move this into its own file so it doesn't clutter the StateContainer.

Reply:
Completed this item in the dowloadAmpHtml branch